b2b-sample/guardio-insurance-sample-application/util/util/apiUtil/decodeMeObject.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2022 WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *http://www.apache.org/licenses/LICENSE-2.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+ export default function decodeMeObject(user) {
+
+    if(user.id==undefined || user.userName==undefined || user.name==undefined || user.emails==undefined ) {
```

Comment:
formatting issue

Reply:
This is fixed with later PR's

### File
- https://github.com/wso2/samples-is/blob/master/b2b-sample/libs/business-app/data-access/data-access-controller/src/lib/controller/controllerMe/controllerDecodeMe.ts

### Commit
- https://github.com/wso2/samples-is/commit/c56439215193067034ac953d02cb38272b2be8d4#diff-41837f5472f12624dbae0a8457f15e50adb73bf6a7f5f2aac6228a4616a1d56f